editor/editor_file_system.cpp

```diff
@@ -2879,6 +2886,33 @@ Error EditorFileSystem::_resource_import(const String &p_path) {
 	return OK;
 }
 
+Ref<Resource> EditorFileSystem::_load_resource_on_startup(ResourceFormatImporter *p_importer, const String &p_path, Error *r_error, bool p_use_sub_threads, float *r_progress, ResourceFormatLoader::CacheMode p_cache_mode) {
+	ERR_FAIL_NULL_V(p_importer, Ref<Resource>());
+
+	if (!FileAccess::exists(p_path)) {
+		ERR_FAIL_V_MSG(Ref<Resource>(), vformat("Failed loading resource: %s. The file doesn't seem to exist.", p_path));
+	}
```

Comment:
Is this still needed/relevant given the added check on `pat.path` in `load_internal`?

It does sound relevant to raise an error here if we're attempting to load a source file which doesn't exist at all, but does that situation actually happen on startup load?

Shouldn't hurt to check for good measure anyway as this situation would be a bug, but if we're concerned about the performance impact of a file existence check, this could be reconsidered.